tests/contrib/mongoengine/test.py

```diff
@@ -193,42 +193,6 @@ def test_analytics_without_rate(self):
             assert len(spans) == 1
             assert spans[0].get_metric(ANALYTICS_SAMPLE_RATE_KEY) == 1.0
 
-    @TracerTestCase.run_in_subprocess(env_overrides=dict(DD_TRACE_SPAN_ATTRIBUTE_SCHEMA="v0"))
-    def test_span_name_v0_schema(self):
```

Comment:
Why are these tests no longer necessary?

Reply:
They still exist, they've just been moved to different UnitTest classes.

The move is because the test agent is looking for the 'default' service name updates on spans  generated with the env/value pair: `DD_TRACE_SPAN_ATTRIBUTE_SCHEMA=v1` and these UnitTest classes use the manual api to set service name, so the test agent fails.